spec/controllers/criteria_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe CriteriaController, type: :controller do
 
-  describe 'Using Flexible criteria' do
+  describe 'Using Rubric criteria' do
```

Comment:
All I can tell about the changes here is that you swapped the order from Flexible -> Rubric to Rubric -> Flexible. This seems strange.